src/serviceWorker.ts

```diff
@@ -0,0 +1,149 @@
+// This optional code is used to register a service worker.
```

Comment:
don't think we need this file since we are not going with a progressive web app (pwa)